scripts/variantstore/wdl/extract/build_base.Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apk add autoconf bash cmake g++ gcc make ninja python3-dev git openssl-dev z
 # including the numpy Python module. The main stage will then use the same base image and copy over the artifacts
 # produced by the build stage without having to install development tools or clean up after a build.
 
-ARG ARROW_VERSION=10.0.0
+ARG ARROW_VERSION=11.0.0
```

Comment:
I was able to upgrade major versions of Arrow and bcftools with no changes other than these 🥳